src/include/catalog/postgres/pg_statistic.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "catalog/catalog_column_def.h"
+#include "catalog/catalog_defs.h"
+
+namespace noisepage::storage {
+class RecoveryManager;
+}  // namespace noisepage::storage
+
+namespace noisepage::catalog::postgres {
+class Builder;
+class PgCoreImpl;
+
+class PgStatistic {
+ public:
+  static constexpr table_oid_t STATISTIC_TABLE_OID = table_oid_t(91);
+  static constexpr index_oid_t STATISTIC_OID_INDEX_OID = index_oid_t(92);
+
+ private:
+  friend class Builder;
+  friend class PgCoreImpl;
+
+  /*
+   * Column names of the form "STA[name]_COL_OID" are present in the PostgreSQL
+   * catalog specification and columns of the form "STA_[name]_COL_OID" are
+   * terrier-specific additions (generally pointers to internal objects).
+   */
+  static constexpr col_oid_t STARELID_COL_OID = col_oid_t(1);     // INTEGER (fkey: pg_class) [table_oid_t]
```

Comment:
Could you look at how `pg_proc` or similar does these now?

Reply:
Done